tss-esapi/src/abstraction/transient/key_attestation.rs

```diff
@@ -188,6 +188,24 @@ impl TransientKeyContext {
             session,
         ))
     }
+
+    pub fn quote(&mut self, key: ObjectWrapper, nonce: Vec<u8>) -> Result<web_authn::TpmPlatStmt> {
+        let key_handle = self.load_key(key.params, key.material, key.auth)?;
```

Comment:
You'll need to flush out this handle after creating the `TpmPlatStmt` (whether it succeeded or not)

Reply:
And same for the keys in `certify`!